### ISSUE TITLE
nebula.info-jar-props doesn't seem to be exported

### ISSUE THREAD

**Issue by jeremyrayner** [nebula-plugins/gradle-info-plugin#27]
Great plugin, I noticed however that 'nebula.info-jar-props' doesn't seem to be available individually, looks like info-jar-props isn't in the gradle-info-plugin build file...  (I run the plugins individually rather than the uber plugin)

Cheers

Jez

**Comment by DanielThomas** [comment]
Yep, looks like we're missing a properties file for that plugin. We'll need to add it.
